src/UNL/VisitorChat/Controller.php

```diff
@@ -53,6 +53,8 @@ class Controller extends \Epoch\Controller
     
     public static $pagetitle = "UNLchat";
     
+    public static $badWords = array('fuck', 'shit', 'pussy', 'ejaculate');
```

Comment:
Haha, good point. I thought a default value would be helpful, but I didn't think it through. :)